### PR TITLE
fix: Use main as default test image

### PR DESCRIPTION
**Pull request by cherrycl**
Revert #881 

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [X] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [X] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

### CODE REVIEW

File: TAF/utils/scripts/snap/snap-utils.sh [hunk 1]
```diff
@@ -78,7 +78,7 @@ snap_taf_update_consul()
 
 snap_set_messagebus_to_mqtt()
 {
-    # note that the integration tests use mqtt as the bus https://github.com/edgexfoundry/edgex-compose/blob/napa/taf/docker-compose-taf-mqtt-bus.yml
+    # note that the integration tests use mqtt as the bus https://github.com/edgexfoundry/edgex-compose/blob/main/taf/docker-compose-taf-mqtt-bus.yml
```
Comment (cloudxxx8) [inline TAF/utils/scripts/snap/snap-utils.sh:81]:
```suggestion
    # note that the integration tests use mqtt as the bus
```

File: docs/edgex-taf.md [hunk 2]
```diff
@@ -128,7 +128,7 @@ To write the automation testing for virtual device service, complete the followi
 
 6. Add the DS to the docker-compose File
 
-    In this document, we deploy all services using docker, so we must add the docker images to the docker-compose file, see example device-virtual as illustrated below: (Full example please reference [edgex-compose](https://github.com/edgexfoundry/edgex-compose/blob/napa/taf/docker-compose-taf-no-secty.yml))
+    In this document, we deploy all services using docker, so we must add the docker images to the docker-compose file, see example device-virtual as illustrated below: (Full example please reference [edgex-compose](https://github.com/edgexfoundry/edgex-compose/blob/main/taf/docker-compose-taf-no-secty.yml))
```
Comment (cloudxxx8) [inline docs/edgex-taf.md:131]:
```suggestion
    In this document, we deploy all services using docker, so we must add the docker images to the docker-compose file, see example device-virtual as illustrated below: (Full example please reference [edgex-compose](https://github.com/edgexfoundry/edgex-compose/blob/main/taf/docker-compose-taf-no-secty.yml), switching to different branch or tag for different versions)
```

**Review by cloudxxx8 — CHANGES_REQUESTED** [review]

File: docs/run-tests-on-local.md [hunk 3]
```diff
@@ -61,13 +61,13 @@ Open the report file by browser: ${WORK_DIR}/TAF/testArtifacts/reports/cp-edgex/
...
     ${USE_SECURITY}: - (false) | -security- (true)
-    ${USE_SHA1}: napa
+    ${USE_SHA1}: main
     ${TEST_STRATEGY}: functional-test | integration-test
 
     # Fetch the latest docker-compose file
     cd ${HOME}/edgex-taf/TAF/utils/scripts/docker
     sh get-compose-file.sh ${USE_ARCH} ${USE_SECURITY} ${USE_SHA1} ${TEST_STRATEGY}
-    # ex. sh get-compose-file.sh x86_64 - napa functional-test
+    # ex. sh get-compose-file.sh x86_64 - main functional-test
```
[the remark this answers is not in the document]
Comment (cherrycl) [reply docs/run-tests-on-local.md:70]: This example is correct, the parameter can't be removed.

File: TAF/config/performance-metrics/configuration.py [hunk 4]
```diff
@@ -15,7 +15,7 @@
 
 
 # Suite: 1_resource_usage
-# Prior release (MINNESOTA) image size (MB) for x86
+# Prior release (NAPA) image size (MB) for x86
```
[the remark this answers is not in the document]
Comment (cherrycl) [reply TAF/config/performance-metrics/configuration.py:18]: Updated

File: TAF/utils/scripts/snap/snap-utils.sh (hunk 1 above)
Comment (cherrycl) [reply TAF/utils/scripts/snap/snap-utils.sh]: Updated

File: docs/edgex-taf.md (hunk 2 above)
Comment (cherrycl) [reply docs/edgex-taf.md]: Updated

**Review by cloudxxx8 — APPROVED** [review]
LGTM

**Pull request merged**